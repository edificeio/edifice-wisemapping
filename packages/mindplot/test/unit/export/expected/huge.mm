<map version="1.0.1">
    <node ID="ID_null" TEXT="I Care">
        <edge COLOR="#121110"/>
        <node ID="ID_null" POSITION="left" STYLE="bubble" TEXT="De markt">
            <richcontent TYPE="NOTE">
                <html xmlns="http://www.w3.org/1999/xhtml">
                    <head><head/>
                    <body>
                        <p/>
                    </body>
                </html>
            </richcontent>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="hygi&#235;ne"/>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="diensten"/>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="energie">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kamer herkent gebruiker"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="automatische lichten"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="inrichting (woning)">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="advies">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kleuren en stijlen"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="monitoring">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="eigendomsverificatie"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kwaliteit"/>
                </node>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="medicijnen">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="alternatieven"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="neveneffecten"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="werking"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kwaliteit"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="electronica">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="monitoring">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="productie verloop">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="testfasen"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wat/wanneer"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="door wie"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="levensloop">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="gebruikscycli"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kwaliteit">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="reparatieservice"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="rechtstreekse link naar producent bij falen"/>
                    </node>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="eigendomsverificatie">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="afdanking (sluikstorten)"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="koop-verkoop"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="GPS tracking"/>
                </node>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="voeding">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="zie bij handelingen --&gt;winkelen"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="rectangle" TEXT="kleding">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="stijlen">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="begeleiding bij het winkelen naar gelijkaardige kleding"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="monitoring">
                    <richcontent TYPE="NOTE">
                        <html xmlns="http://www.w3.org/1999/xhtml">
                            <head><head/>
                            <body>
                                <p/>
                            </body>
                        </html>
                    </richcontent>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kwaliteitsinfo bij aankoop">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="reviews"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="info uit cloud"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="informatiebronnen per merk">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="slimme kledij"/>
                </node>
            </node>
        </node>
        <node ID="ID_null" POSITION="right" STYLE="fork" TEXT="veiligheid">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="criminaliteit">
                <richcontent TYPE="NOTE">
                    <html xmlns="http://www.w3.org/1999/xhtml">
                        <head><head/>
                        <body>
                            <p/>
                        </body>
                    </html>
                </richcontent>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Aantal ongevallen/criminele feiten registreren">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Terugkoppeling naar politie/verkeersdienst"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Meer controle"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Mapping van probleemsituaties"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="diefstal">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="tracking via gps"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wapenhandel">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="opsporen"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="drugs">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="opsporen">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="testen per woonregio (via afvoerstelsels)"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="individuele tests"/>
                    </node>
                </node>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="verkeer">
                <richcontent TYPE="NOTE">
                    <html xmlns="http://www.w3.org/1999/xhtml">
                        <head><head/>
                        <body>
                            <p/>
                        </body>
                    </html>
                </richcontent>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="ongevallen">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="info over de staat van het wegdek">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aanpassing van max. snelheid"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="waarschuwingen"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wegenwerken"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="oorzaak">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="opsporen van vluchtmisdrijf"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Automatisch opbellen van hulpdiensten"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="nagaan van de oorzaak"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="preventie">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bestraffen van chauffeurs?"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="roekeloos rijgedrag opsporen">
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="via sensoren aan boord versnelling, manouvres, snelheid,... registreren"/>
                        </node>
                    </node>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Openbare fietsen">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Fiets bestellen in grote steden"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Trackingsysteem, # fietsen op 1 plaats"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Openbaar vervoer"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="filevorming">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Omleiding zoeken"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="preventie">
                        <richcontent TYPE="NOTE">
                            <html xmlns="http://www.w3.org/1999/xhtml">
                                <head><head/>
                                <body>
                                    <p/>
                                </body>
                            </html>
                        </richcontent>
                    </node>
                </node>
            </node>
        </node>
        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="handelingen">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="winkelen">
                <font BOLD="true" SIZE="12"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="winkelhulp">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="boodschappenlijst">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="alternatieven indien uitverkocht"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="dichtste bij bovenaan"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="recepten generator">
                        <richcontent TYPE="NOTE">
                            <html xmlns="http://www.w3.org/1999/xhtml">
                                <head><head/>
                                <body>
                                    <p/>
                                </body>
                            </html>
                        </richcontent>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bestellingen op afstand"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="digitale portemonnee"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="productinfo">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="alternatieve producten"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="di&#235;et checker"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kwaliteit">
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="voedingstoffen"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bio"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="databases v. reviews"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="via cloud"/>
                        </node>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="prijs"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="allergie">
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Salmonella sensor"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Sensor voor vers fruit en vlees"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="Gezondheidscontrole"/>
                        </node>
                    </node>
                </node>
            </node>
        </node>
        <node ID="ID_null" POSITION="right" STYLE="fork" TEXT="industrie"/>
        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aandoeningen en situaties">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="dementen">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="eten"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wassen"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wegloopdetectie"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="zwangerschap">
                <font BOLD="true" SIZE="12"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="geboortetimer">
                    <richcontent TYPE="NOTE">
                        <html xmlns="http://www.w3.org/1999/xhtml">
                            <head><head/>
                            <body>
                                <p/>
                            </body>
                        </html>
                    </richcontent>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="info over voeding">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="via cloud info over voeding">
                        <richcontent TYPE="NOTE">
                            <html xmlns="http://www.w3.org/1999/xhtml">
                                <head><head/>
                                <body>
                                    <p/>
                                </body>
                            </html>
                        </richcontent>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wat is gezond"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="baby monitoren"/>
            </node>
        </node>
        <node ID="ID_null" POSITION="right" STYLE="fork" TEXT="sport">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="opvolging van de sporter">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="gps"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="cadans"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="energieverbruik"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bloeddruk"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="hartslag"/>
            </node>
        </node>
        <node ID="ID_null" POSITION="right" STYLE="fork" TEXT="cultuur">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="festival"/>
        </node>
        <node ID="ID_null" POSITION="right" STYLE="fork" TEXT="bouw">
            <richcontent TYPE="NOTE">
                <html xmlns="http://www.w3.org/1999/xhtml">
                    <head><head/>
                    <body>
                        <p/>
                    </body>
                </html>
            </richcontent>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="opvolging zoals plannen zijn getekend">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="nameten"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="juistheid van materialen"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="isolatie">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="warmtemetingen"/>
                </node>
            </node>
        </node>
        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="ziektes">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="stress">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="monitoring">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="in welke afdeling een probleem">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aanpak"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aan welke factoren ligt dat"/>
                    </node>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="afreageren"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="thuisverzorging">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="link naar ziekenhuis"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="link naar dokter"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="meting van de symptomen">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="objectieve metingen"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="toevoeging van eigen waarneming"/>
                </node>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="verschillende ziektes">
                <richcontent TYPE="NOTE">
                    <html xmlns="http://www.w3.org/1999/xhtml">
                        <head><head/>
                        <body>
                            <p/>
                        </body>
                    </html>
                </richcontent>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="anorexia">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="eten ze voldoende"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="nagaan of ze eten"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="astma">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="detectie van luchtkwaliteit">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="in kaart brengen van">
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="allergie veroorzakende deeltjes"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="pollen"/>
                            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="vervuiling"/>
                        </node>
                    </node>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="epilepsie">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aanval voorspellen??"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="diabetes">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="suikerspiegel meten">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="automatische inspuiting"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="alert indien te laag"/>
                    </node>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aids">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="vergroten van de database aan info"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="mucoviscidose">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="alert voor donor"/>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="longcapaciteit meten"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="MS">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bevorderen van communicatie tijdens de aftakeling">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="naar familie en vrienden toe"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="naar de dokters toe"/>
                    </node>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="feedback van de lichaamsconditie">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aan dokter">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="op afstand consulatie"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="volledig overzicht"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="contacteert patient indien nodig"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="link">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="naar behandelend arts"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="naar hulpdiensten"/>
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="naar ziekenhuis"/>
                    </node>
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="aan de pati&#235;nt">
                        <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="waarschuwingen"/>
                    </node>
                </node>
            </node>
        </node>
        <node ID="ID_null" POSITION="right" STYLE="fork" TEXT="milieu">
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="alternatieve energie">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="kernenergie"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="zon"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="wind"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="waterreserves">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="vervuiling"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="reserves"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="eco systemen">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="diversiteit"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="dioxines"/>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="afval">
                <node ID="ID_null" POSITION="left" STYLE="fork"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="cradle 2 cradle"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="riolen"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="verwerking"/>
            </node>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="pollutie"/>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="mag ik dit door mijn gootsteen kappen"/>
            <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bosbouw">
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="bodemvervuiling">
                    <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="onderzoek"/>
                </node>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="parasieten"/>
                <node ID="ID_null" POSITION="left" STYLE="fork" TEXT="ziektes bij bomen"/>
            </node>
        </node>
    </node>
</map>