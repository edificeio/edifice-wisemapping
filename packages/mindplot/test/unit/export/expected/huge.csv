I Care 
,
            De markt 
,
            ,
            hygiëne 
,
            ,
            diensten 
,
            ,
            energie 
,
            ,
            ,
            kamer herkent gebruiker 
,
            ,
            ,
            automatische lichten 
,
            ,
            inrichting (woning) 
,
            ,
            ,
            advies 
,
            ,
            ,
            ,
            kleuren en stijlen 
,
            ,
            ,
            monitoring 
,
            ,
            ,
            ,
            eigendomsverificatie 
,
            ,
            ,
            ,
            kwaliteit 
,
            ,
            medicijnen 
,
            ,
            ,
            alternatieven 
,
            ,
            ,
            neveneffecten 
,
            ,
            ,
            werking 
,
            ,
            ,
            kwaliteit 
,
            ,
            electronica 
,
            ,
            ,
            monitoring 
,
            ,
            ,
            ,
            productie verloop 
,
            ,
            ,
            ,
            ,
            testfasen 
,
            ,
            ,
            ,
            ,
            wat/wanneer 
,
            ,
            ,
            ,
            ,
            door wie 
,
            ,
            ,
            ,
            levensloop 
,
            ,
            ,
            ,
            ,
            gebruikscycli 
,
            ,
            ,
            ,
            kwaliteit 
,
            ,
            ,
            ,
            ,
            reparatieservice 
,
            ,
            ,
            ,
            ,
            rechtstreekse link naar producent bij falen 
,
            ,
            ,
            eigendomsverificatie 
,
            ,
            ,
            ,
            afdanking (sluikstorten) 
,
            ,
            ,
            ,
            koop-verkoop 
,
            ,
            ,
            ,
            GPS tracking 
,
            ,
            voeding 
,
            ,
            ,
            zie bij handelingen -->winkelen 
,
            ,
            kleding 
,
            ,
            ,
            stijlen 
,
            ,
            ,
            ,
            begeleiding bij het winkelen naar gelijkaardige kleding 
,
            ,
            ,
            monitoring 
,
            ,
            ,
            kwaliteitsinfo bij aankoop 
,
            ,
            ,
            ,
            reviews 
,
            ,
            ,
            ,
            info uit cloud 
,
            ,
            ,
            informatiebronnen per merk 
,
            ,
            ,
            ,
            slimme kledij 
,
            veiligheid 
,
            ,
            criminaliteit 
,
            ,
            ,
            Aantal ongevallen/criminele feiten registreren 
,
            ,
            ,
            ,
            Terugkoppeling naar politie/verkeersdienst 
,
            ,
            ,
            ,
            Meer controle 
,
            ,
            ,
            ,
            Mapping van probleemsituaties 
,
            ,
            ,
            diefstal 
,
            ,
            ,
            ,
            tracking via gps 
,
            ,
            ,
            wapenhandel 
,
            ,
            ,
            ,
            opsporen 
,
            ,
            ,
            drugs 
,
            ,
            ,
            ,
            opsporen 
,
            ,
            ,
            ,
            ,
            testen per woonregio (via afvoerstelsels) 
,
            ,
            ,
            ,
            ,
            individuele tests 
,
            ,
            verkeer 
,
            ,
            ,
            ongevallen 
,
            ,
            ,
            ,
            info over de staat van het wegdek 
,
            ,
            ,
            ,
            ,
            aanpassing van max. snelheid 
,
            ,
            ,
            ,
            ,
            waarschuwingen 
,
            ,
            ,
            ,
            ,
            wegenwerken 
,
            ,
            ,
            ,
            oorzaak 
,
            ,
            ,
            ,
            ,
            opsporen van vluchtmisdrijf 
,
            ,
            ,
            ,
            ,
            Automatisch opbellen van hulpdiensten 
,
            ,
            ,
            ,
            ,
            nagaan van de oorzaak 
,
            ,
            ,
            ,
            preventie 
,
            ,
            ,
            ,
            ,
            bestraffen van chauffeurs? 
,
            ,
            ,
            ,
            ,
            roekeloos rijgedrag opsporen 
,
            ,
            ,
            ,
            ,
            ,
            via sensoren aan boord versnelling, manouvres, snelheid,... registreren 
,
            ,
            ,
            Openbare fietsen 
,
            ,
            ,
            ,
            Fiets bestellen in grote steden 
,
            ,
            ,
            ,
            Trackingsysteem, # fietsen op 1 plaats 
,
            ,
            ,
            Openbaar vervoer 
,
            ,
            ,
            filevorming 
,
            ,
            ,
            ,
            Omleiding zoeken 
,
            ,
            ,
            ,
            preventie 
,
            handelingen 
,
            ,
            winkelen 
,
            ,
            ,
            winkelhulp 
,
            ,
            ,
            ,
            boodschappenlijst 
,
            ,
            ,
            ,
            ,
            alternatieven indien uitverkocht 
,
            ,
            ,
            ,
            ,
            dichtste bij bovenaan 
,
            ,
            ,
            ,
            recepten generator 
,
            ,
            ,
            ,
            bestellingen op afstand 
,
            ,
            ,
            ,
            digitale portemonnee 
,
            ,
            ,
            ,
            productinfo 
,
            ,
            ,
            ,
            ,
            alternatieve producten 
,
            ,
            ,
            ,
            ,
            diëet checker 
,
            ,
            ,
            ,
            ,
            kwaliteit 
,
            ,
            ,
            ,
            ,
            ,
            voedingstoffen 
,
            ,
            ,
            ,
            ,
            ,
            bio 
,
            ,
            ,
            ,
            ,
            ,
            databases v. reviews 
,
            ,
            ,
            ,
            ,
            ,
            via cloud 
,
            ,
            ,
            ,
            ,
            prijs 
,
            ,
            ,
            ,
            ,
            allergie 
,
            ,
            ,
            ,
            ,
            ,
            Salmonella sensor 
,
            ,
            ,
            ,
            ,
            ,
            Sensor voor vers fruit en vlees 
,
            ,
            ,
            ,
            ,
            ,
            Gezondheidscontrole 
,
            industrie 
,
            aandoeningen en situaties 
,
            ,
            dementen 
,
            ,
            ,
            eten 
,
            ,
            ,
            wassen 
,
            ,
            ,
            wegloopdetectie 
,
            ,
            zwangerschap 
,
            ,
            ,
            geboortetimer 
,
            ,
            ,
            info over voeding 
,
            ,
            ,
            ,
            via cloud info over voeding 
,
            ,
            ,
            ,
            wat is gezond 
,
            ,
            ,
            baby monitoren 
,
            sport 
,
            ,
            opvolging van de sporter 
,
            ,
            ,
            gps 
,
            ,
            ,
            cadans 
,
            ,
            ,
            energieverbruik 
,
            ,
            ,
            bloeddruk 
,
            ,
            ,
            hartslag 
,
            cultuur 
,
            ,
            festival 
,
            bouw 
,
            ,
            opvolging zoals plannen zijn getekend 
,
            ,
            ,
            nameten 
,
            ,
            ,
            juistheid van materialen 
,
            ,
            ,
            isolatie 
,
            ,
            ,
            ,
            warmtemetingen 
,
            ziektes 
,
            ,
            stress 
,
            ,
            ,
            monitoring 
,
            ,
            ,
            ,
            in welke afdeling een probleem 
,
            ,
            ,
            ,
            ,
            aanpak 
,
            ,
            ,
            ,
            ,
            aan welke factoren ligt dat 
,
            ,
            ,
            afreageren 
,
            ,
            thuisverzorging 
,
            ,
            ,
            link naar ziekenhuis 
,
            ,
            ,
            link naar dokter 
,
            ,
            ,
            meting van de symptomen 
,
            ,
            ,
            ,
            objectieve metingen 
,
            ,
            ,
            ,
            toevoeging van eigen waarneming 
,
            ,
            verschillende ziektes 
,
            ,
            ,
            anorexia 
,
            ,
            ,
            ,
            eten ze voldoende 
,
            ,
            ,
            ,
            nagaan of ze eten 
,
            ,
            ,
            astma 
,
            ,
            ,
            ,
            detectie van luchtkwaliteit 
,
            ,
            ,
            ,
            ,
            in kaart brengen van 
,
            ,
            ,
            ,
            ,
            ,
            allergie veroorzakende deeltjes 
,
            ,
            ,
            ,
            ,
            ,
            pollen 
,
            ,
            ,
            ,
            ,
            ,
            vervuiling 
,
            ,
            ,
            epilepsie 
,
            ,
            ,
            ,
            aanval voorspellen?? 
,
            ,
            ,
            diabetes 
,
            ,
            ,
            ,
            suikerspiegel meten 
,
            ,
            ,
            ,
            ,
            automatische inspuiting 
,
            ,
            ,
            ,
            ,
            alert indien te laag 
,
            ,
            ,
            aids 
,
            ,
            ,
            ,
            vergroten van de database aan info 
,
            ,
            ,
            mucoviscidose 
,
            ,
            ,
            ,
            alert voor donor 
,
            ,
            ,
            ,
            longcapaciteit meten 
,
            ,
            ,
            MS 
,
            ,
            ,
            ,
            bevorderen van communicatie tijdens de aftakeling 
,
            ,
            ,
            ,
            ,
            naar familie en vrienden toe 
,
            ,
            ,
            ,
            ,
            naar de dokters toe 
,
            ,
            ,
            feedback van de lichaamsconditie 
,
            ,
            ,
            ,
            aan dokter 
,
            ,
            ,
            ,
            ,
            op afstand consulatie 
,
            ,
            ,
            ,
            ,
            volledig overzicht 
,
            ,
            ,
            ,
            ,
            contacteert patient indien nodig 
,
            ,
            ,
            ,
            link 
,
            ,
            ,
            ,
            ,
            naar behandelend arts 
,
            ,
            ,
            ,
            ,
            naar hulpdiensten 
,
            ,
            ,
            ,
            ,
            naar ziekenhuis 
,
            ,
            ,
            ,
            aan de patiënt 
,
            ,
            ,
            ,
            ,
            waarschuwingen 
,
            milieu 
,
            ,
            alternatieve energie 
,
            ,
            ,
            kernenergie 
,
            ,
            ,
            zon 
,
            ,
            ,
            wind 
,
            ,
            waterreserves 
,
            ,
            ,
            vervuiling 
,
            ,
            ,
            reserves 
,
            ,
            eco systemen 
,
            ,
            ,
            diversiteit 
,
            ,
            dioxines 
,
            ,
            afval 
,
            ,
            ,
             
,
            ,
            ,
            cradle 2 cradle 
,
            ,
            ,
            riolen 
,
            ,
            ,
            verwerking 
,
            ,
            pollutie 
,
            ,
            mag ik dit door mijn gootsteen kappen 
,
            ,
            bosbouw 
,
            ,
            ,
            bodemvervuiling 
,
            ,
            ,
            ,
            onderzoek 
,
            ,
            ,
            parasieten 
,
            ,
            ,
            ziektes bij bomen 
